app/services/diagnostics/move_inspector.rb

```diff
@@ -51,6 +54,7 @@ def generate
       if move.journeys.any?
         @output << "#{'ID'.ljust(37)}\t#{'TIMESTAMP'.ljust(27)}\t#{'STATE'.ljust(12)}\t#{'BILLABLE'.ljust(9)}\t#{'SUPPLIER'.ljust(9)}\tFROM --> TO\n"
         move.journeys.default_order.each do |journey| # NB use each to preserve sort order
+          # NB only show event details if include_person_details==true, as they could contain personal details
```

Comment:
I can't see the code that this comment relates to - this block doesn't seem to be checking the value of `include_person_details`.

Reply:
oops, the comment was meant for the previous block, now fixed